### ISSUE TITLE
PIXI.js v5 / Attempted import error: 'extras' is not exported from 'pixi.js'

### ISSUE THREAD

**Issue by fabienjuif** [inlet/react-pixi#115]
### Description
Hey :wave:,

I wanted to bump PIXI.js to the v5 and I got this error at compilation:
```
../node_modules/@inlet/react-pixi/dist/react-pixi.module-dev.js
Attempted import error: 'extras' is not exported from 'pixi.js'.
```

I saw this issue: https://github.com/inlet/react-pixi/issues/98
It seems you fixed the issue and other issues related to PIXI.js v4 in last version: https://github.com/inlet/react-pixi/releases/tag/v0.7.1

But you can see, I still have the issue from #98. 
Do you have an idea to get rid of it?

### Steps to reproduce

1. Use CRA: `yarn create react-app .`
2. Install PIXI.js v5.0.4: `yarn add pixi.js@5.0.4`
3. Install @inlet/react-pixi v0.7.1: `yarn add @inlet/react-pixi@0.7.1`
4. Paste this code as replacement of _App.js_:
```jsx
import React from 'react';
import { Stage, Text } from '@inlet/react-pixi'

const App = () => (
  <Stage>
    <Text text="Hey!" />
  </Stage>
);

export default App;
```
5. Start the app: `yarn start`

### Additional info

- `@inlet/react-pixi` version: 0.7.1
- `React` version: 16.8.6
- `ReactDOM` version: 16.8.6
- `PIXI` version: 5.0.4

**Comment by inlet** [comment]
Hi @fabienjuif,

Thanks! It tries to resolve the `pixi.js v4` components which have different namespaces (note the `extras` error message). To solve this (just a temporary fix though) you can force `create-react-app` to use the commonjs import:

```js
import { Stage } from '@inlet/react-pixi/index'
``` 

However, you won't be able to use the `Stage` object either as this uses `pixi.js v4` `PIXI.Application` which is not compatible with the new `v5` API. However you can easily create your own Stage object.

Even better would be to create a PR and upgrade `Stage` and `components/*` to make it compatible with `PIXI v5`. 🙏

**Comment by inlet** [comment]
I can do this as well, but haven't got time the coming weeks (working on some projects). If you need it now we can work on it together. Just make a PR and I'll contribute to it.

**Comment by fabienjuif** [comment]
Hey @inlet I try to take a look then, but without promising anything :)

**Comment by Adamfsk** [comment]
@fabienjuif This is caused by the old typings for Pixi in your v5 branch. My PR I will send through to you shortly will fix it! Just testing now to make sure it works!

**Comment by inlet** [comment]
Fixed in #116

**Comment by darklightblue** [comment]
Hi, so how do you use it? I'm trying to play around with this https://codepen.io/inlet/pen/5aaed9ddeadb4d7bb5ea435d77df397a but it seems that it is still using v4.
